app/views/browse/build-config.html

```diff
@@ -75,8 +75,8 @@
             <div class="row" ng-if="loaded">
               <div class="col-md-12" ng-class="{ 'hide-tabs' : !buildConfig }">
                 <uib-tabset>
-                  <uib-tab active="selectedTab.summary">
-                    <uib-tab-heading>Summary</uib-tab-heading>
+                  <uib-tab active="selectedTab.builds">
```

Comment:
The link works and shows the first tab, but it doesn't update the param in the URL. I can change that if you like.